### PR TITLE
DBTP-590 - Update OpenSearch template to allow scaling without instance replacement

### PR DESCRIPTION
**Pull request by JohnStainsby**
- Update OpenSearch template to change specific properties rather than conditionally defining a separate instance. This allows the instance count to change without CloudFormation replacing the resource
- Bump version number

### CODE REVIEW

File: dbt_copilot_helper/templates/addons/env/opensearch.yml [hunk 1]
```diff
@@ -86,10 +85,9 @@ Resources:
       ToPort: 443
       SourceSecurityGroupId: !Ref EnvironmentSecurityGroup
 
-  # Single opensearch instance
+  # Opensearch
```
Comment (yusufsheiqh) [inline dbt_copilot_helper/templates/addons/env/opensearch.yml:88]: This comment seems a bit redundant to me, as this is the OpenSearch addon template.

```suggestion
```

File: tests/copilot_helper/fixtures/make_addons/expected/environments/addons/my-opensearch.yml [hunk 2]
```diff
@@ -86,10 +85,9 @@ Resources:
       ToPort: 443
       SourceSecurityGroupId: !Ref EnvironmentSecurityGroup
 
-  # Single opensearch instance
+  # Opensearch
```
Comment (yusufsheiqh) [inline tests/copilot_helper/fixtures/make_addons/expected/environments/addons/my-opensearch.yml:88]:
```suggestion
```

File: tests/copilot_helper/fixtures/make_addons/expected/environments/addons/my-opensearch-longer.yml [hunk 3]
```diff
@@ -86,10 +85,9 @@ Resources:
       ToPort: 443
       SourceSecurityGroupId: !Ref EnvironmentSecurityGroup
 
-  # Single opensearch instance
+  # Opensearch
```
Comment (yusufsheiqh) [inline tests/copilot_helper/fixtures/make_addons/expected/environments/addons/my-opensearch-longer.yml:88]:
```suggestion
```

**Review by yusufsheiqh — COMMENTED** [review]
Nice work.

**Review by WillGibson — APPROVED** [review]
Thought I had approved this yesterday...

**Pull request merged**